### PR TITLE
Show number of nodes tracking a repo

### PR DESCRIPTION
**Pull request by xphoniex**
Review but please wait for the merge and deployment of the corresponding patch on API side.

Also, as per cloudhead's request, "Tracked by {trackings} nodes" message will be replace by network icon and count.

Screenshot from 2023-05-31 20-47-15

### CODE REVIEW

File: src/views/projects/Header.svelte [hunk 1]
```diff
@@ -116,4 +119,9 @@
       {baseUrl.hostname}
     </SquareButton>
   </Link>
+  <SquareButton>
```
Comment (sebastinez) [inline src/views/projects/Header.svelte:122]: I would add here that `hoverable={false}` so we match the style of the `contributors` label

File: tests/e2e/project.spec.ts [hunk 2]
```diff
@@ -64,6 +64,9 @@ test("navigate to project", async ({ page }) => {
 
   // Show rendered README.md contents.
   await expect(page.locator("text=Git test repository")).toBeVisible();
+
+  // Number of nodes tracking this project.
+  await expect(page.locator("text=3 nodes")).toBeVisible();
```
Comment (xphoniex) [inline tests/e2e/project.spec.ts:69]: added this, wdyt @sebastinez ? can't run it locally to be sure

Comment (sebastinez) [reply tests/e2e/project.spec.ts:69]: Yeah that works, to get the CI to pass you'll need to also update the `heartwood-version` file

```diff
diff --git a/tests/support/heartwood-version b/tests/support/heartwood-version
index 81e79f2..ce545a9 100644
--- a/tests/support/heartwood-version
+++ b/tests/support/heartwood-version
@@ -1 +1 @@
-97f122c4512fbc5ea0b04a11db01d0d2d0977527
+fd91cf4d28bd370a6c5beb9ca158e4d91432df25
```

File: src/views/projects/Header.svelte [hunk 3]
```diff
@@ -19,6 +20,8 @@
 
   export let openPatchCount: number;
   export let openIssueCount: number;
+
```
Comment (sebastinez) [inline src/views/projects/Header.svelte:23]: Can you remove this line? So the props are in one block.

File: src/views/projects/Header.svelte [hunk 4]
```diff
@@ -4,12 +4,13 @@
...
   import { config } from "@app/lib/config";
   import { pluralize } from "@app/lib/pluralize";
+  import { isLocal } from "@app/lib/utils";
 
   import CloneButton from "@app/views/projects/CloneButton.svelte";
   import Link from "@app/components/Link.svelte";
   import ProjectLink from "@app/components/ProjectLink.svelte";
   import SquareButton from "@app/components/SquareButton.svelte";
-  import { isLocal } from "@app/lib/utils";
+  import Icon from "@app/components/Icon.svelte";
```
Comment (sebastinez) [inline src/views/projects/Header.svelte:13]: We sort the imports by the vim lexical sort, selecting the imports in visual mode and doing a `:sort` but you can also just sort it alphabetically here.

**Review by sebastinez — COMMENTED** [review]
Just saw two nits that would be nice to change before merge

File: src/views/projects/Header.svelte (hunk 4 above)
Comment (xphoniex) [reply src/views/projects/Header.svelte]: [diff](https://github.com/radicle-dev/radicle-interface/compare/66d3e1496e2f5179e58be4af261c99494752b2d0..c3346575eece8c059370a629cea73470ae16ce78)

File: src/views/projects/Header.svelte (hunk 3 above)
Comment (xphoniex) [reply src/views/projects/Header.svelte]: [diff](https://github.com/radicle-dev/radicle-interface/compare/66d3e1496e2f5179e58be4af261c99494752b2d0..c3346575eece8c059370a629cea73470ae16ce78)

**Review by sebastinez — APPROVED** [review]
💯

**Comment by xphoniex** [comment]
Screenshot-from-2023-06-02-15-36-14

**Pull request merged**